packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -138,6 +145,7 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 			outlineMode: hasTemplate,
 			__experimentalCreatePageEntity: createPageEntity,
 			__experimentalUserCanCreatePages: userCanCreatePages,
+			pageOnFront,
```

Comment:
@adamziel Would you mind confidence checking me here? A previous PR I created introduced a typing performance degradation because the `useMemo` was recomputed too frequently. Do you see any possibility of this change causing that?

Reply:
[This one](https://github.com/WordPress/gutenberg/pull/35083#discussion_r723581566) (owch!)

Reply:
I commented with my assessment there. `pageOnFront` seems to be be safe as it is a number, not a function (As long as it's not a `NaN`). Full disclosure, I didn't test it before writing this comment.